src/components/Icons/Medias/VolumeUp.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function VolumeUp({
+  id = 'VolumeUpIcon',
```

Comment:
:pick: This name could be VolumeOn

```suggestion
function VolumeOn({
  id = 'VolumeOnIcon',
```

Reply:
comment resolved in https://github.com/magnetis/astro-native/pull/590/commits/6c0bae3a59e4a2140bd1b1f9ed10a093bae783a3